### ISSUE TITLE
web fallback for not native eligibility stuck in processing

### ISSUE THREAD

**Issue by nerocristallo** [paypal/android-checkout-sdk#153]
I am testing the web fallback for not native eligibility setting the shouldFailEligibility flag in SettingsConfig.

```
SettingsConfig settingsConfig = new SettingsConfig(BuildConfig.DEBUG, true);

CheckoutConfig payPalCheckoutConfig = new CheckoutConfig(this, "XXXXXXXXXXXXX", Environment.SANDBOX, CurrencyCode.USD,UserAction.PAY_NOW, PaymentButtonIntent.CAPTURE, settingsConfig);
```

![Screenshot_20220930-095244](https://user-images.githubusercontent.com/25849016/193224104-e5acf9f6-1b32-49ce-8e94-292ddd00cb44.jpg)

The Web UI gets stuck during processing and the OnApprove callback is not called.

The checkout procedure works correctly when shouldFailEligibility is set to false.

 - Android Version: API 33
 - Device: Pixel 4a
 - SDK Version 0.8.1
 - Client ID "AUVp4x9jsUGfrUszFiMhXMgioo5n875YqlVz4hVrMdp5mBuoigfs7nRIpjZw9wJsJ0afIA3x9gjlk5IU"

NOTE:
I am using PayPal Commerce Platform for Marketplaces and Platforms

**Comment by gabo-alv** [comment]
This got fixed in version `0.8.5`. Please update your SDK to the latest  version and this issue should be solved.

**Comment by nerocristallo** [comment]
I can confirm that it is fixed in 0.8.5

Thank you
